truss/templates/vllm/vllm.Dockerfile.jinja

```diff
@@ -15,6 +15,9 @@ ENV HUGGING_FACE_HUB_TOKEN {{hf_access_token}}
 {%- if hf_cache %}
 COPY ./data /app/data
 COPY ./cache_requirements.txt /app/cache_requirements.txt
+RUN curl -s https://baseten-public.s3.us-west-2.amazonaws.com/bin/b10cp-0.5.4-linux-amd64 -o /app/b10cp; chmod +x /app/b10cp
```

Comment:
`s3://baseten-public/bin/b10cp-5fe8dc7da-linux-amd64` -- use newer version

Reply:
also, let's please add this to the other dockerfiles as well